CONTRIBUTING.md

```diff
@@ -6,3 +6,6 @@ Agreement](https://cla-assistant.io/redfin/react-server). The purpose
 of this license is to protect contributors, Redfin, as well as users
 of this software project. Signing this agreement does not affect your
 rights to use your contributions for any other purpose.
+
```

Comment:
I think we should give other people, who are not watching this repository, a chance to weigh in on the code of conduct before we merge.